packages/analytics/amplify_analytics_pinpoint_dart/pubspec.yaml

```diff
@@ -0,0 +1,39 @@
+name: amplify_analytics_pinpoint_dart
+description: A Dart-only implementation of the Amplify Analytics plugin for Pinpoint.
+version: 0.1.0
+homepage: https://docs.amplify.aws/lib/q/platform/flutter/
+repository:
+issue_tracker: https://github.com/aws-amplify/amplify-flutter/issues
+
+environment:
+  sdk: ">=2.17.0 <3.0.0"
+
+dependencies:
+  amplify_core: ^1.0.0-next.0
+  amplify_secure_storage_dart: ^0.1.0
+  aws_common: ^0.2.2
+  aws_signature_v4: ^0.1.0
+  built_collection: ^5.0.0
+  built_value: ^8.4.0
+  meta: ^1.7.0
+  smithy: ^0.1.0
+  smithy_aws: ^0.1.0
+  uuid: ^3.0.0
+  path: ^1.8.1 # path manipulation
+  drift: ^2.1.0 # local database
+
+  intl: ^0.17.0 # for time format
```

Comment:
Sort these. No need to include the comments.

Reply:
We have a lint for this. If it is not showing in your IDE, see if you can configure it to show.

Reply:
why would we remove comments?  Perhaps not for path, but the inclusion of intl isn't clear.